cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/token/AccessTokenImpl.java

```diff
@@ -93,12 +105,10 @@ public AccessTokenImpl(final String id, final TokenType type, final String clien
     }
 
     @Override
-    @Transient
```

Comment:
This is what was causing the error. Although this looks like a getter but there is no field for `ticket` at all. Not sure how `@Transient` apply to an non-getter.